decouple.py

```diff
@@ -35,22 +35,9 @@ class Config(object):
     """
     Handle .env file format used by Foreman.
     """
-    _BOOLEANS = {'1': True, 'yes': True, 'true': True, 'on': True,
-                 '0': False, 'no': False, 'false': False, 'off': False, '': False}
-
     def __init__(self, repository):
         self.repository = repository
 
-    def _cast_boolean(self, value):
-        """
-        Helper to convert config values to boolean as ConfigParser do.
-        """
-        value = str(value)
-        if value.lower() not in self._BOOLEANS:
-            raise ValueError('Not a boolean: %s' % value)
-
-        return self._BOOLEANS[value.lower()]
```

Comment:
[strtobool](https://docs.python.org/3.7/distutils/apiref.html#distutils.util.strtobool) return 0 and 1. for backward compatibility, should we convert to bool?

Reply:
or change the internal api to use `strtobool` when` cast` is `bool`

@henriquebastos